### PR TITLE
[IA-2168] Enable installing specific chart versions

### PR DESCRIPTION
**Pull request by kyuksel**
https://broadworkbench.atlassian.net/browse/IA-2168

### CODE REVIEW

**Review by Qi77Qi — APPROVED** [review]
probably need to bump version to rc2....otherwise lgtm

File: helm-go-lib/main.go [hunk 1]
```diff
@@ -24,14 +24,14 @@ import (
...
+			my-namespace my-kube-token-string https://12.34.567.890 ./ca_file bitnami/nginx 6.0.5 key1=v1,key2.key3=v2
 */
 func main() {
 	flag.Parse()
 	progArgs := flag.Args()
 	lenProgArgs := len(progArgs)
 
-	if (lenProgArgs != 6) && (lenProgArgs != 7) {
+	if (lenProgArgs != 7) && (lenProgArgs != 8) {
 		fmt.Println("Expected args: <namespace> <kube token> <api server> <ca file> <release name> <chart name> <values>",
```
Comment (jdcanas) [inline helm-go-lib/main.go:35]: should this print be updated too? I don't see anything indicated that theres an optional param

Comment (kyuksel) [reply helm-go-lib/main.go:35]: Yes! Thanks for the catch.

**Comment by kyuksel** [comment]
> probably need to bump version to rc2....otherwise lgtm

Yea, I was going to ask about that. The last PR didn't bump it to RC2 but I see Leo point to an existing RC2...so I'll bump it to RC3.

**Comment by kyuksel** [comment]
@jdcanas @Qi77Qi Thanks for the feedback. Just addressed them.

File: build.sbt [hunk 2]
```diff
@@ -7,7 +7,7 @@ lazy val root = (project in file("."))
   .settings(
     organization := "org.broadinstitute.dsp",
     name := "helm-scala-sdk",
-    version := "0.0.1-RC1",
+    version := "0.0.1-RC3",
```
Comment (rtitle) [inline build.sbt:10]: Maybe need to rebase this PR? I had changed the version to RC2, weird it's not showing in this diff

File: src/test/scala/org/broadinstitute/dsp/HelmManualTest.scala [hunk 3]
```diff
@@ -11,7 +11,7 @@ import cats.effect.IO
  * Once inside sbt shell:
  *    val namespace = ...; val token = ...; val apiServer = ...; val caCertFile = ...;
  *    val test = new org.broadinstitute.dsp.HelmManualTest(namespace, token, apiServer, caCertFile)
- *    test.installChart(release, chart, values)
+ *    test.installChart(release, chartName, values)
```
Comment (rtitle) [inline src/test/scala/org/broadinstitute/dsp/HelmManualTest.scala:14]: `test.installChart(release, chartName, chartVersion, values)`?

File: src/main/scala/org/broadinstitute/dsp/HelmAlgebra.scala [hunk 4]
```diff
@@ -9,7 +9,8 @@ import scala.util.control.NoStackTrace
 trait HelmAlgebra[F[_]] {
   def installChart(
     release: Release,
-    chart: Chart,
+    chartName: ChartName,
+    chartVersion: ChartVersion,
```
Comment (rtitle) [inline src/main/scala/org/broadinstitute/dsp/HelmAlgebra.scala:13]: Was wondering if we should make the version default to latest if not specified? I suppose it makes sense to enforce always setting the version in the library if that's what Leo is going to do.

File: build.sbt (hunk 2 above)
Comment (rtitle) [reply build.sbt:10]: Ohh weird it's RC1 in master branch. Did I forget to push? 🤦 

This looks fine then, I think.

**Review by rtitle — APPROVED** [review]
Looks good

Just as a reminder you need to `sbt ++publish` manually after merging since there is no CI for this repo.

File: build.sbt (hunk 2 above)
Comment (rtitle) [reply build.sbt:10]: Yeah :( sorry

```
rtitle@helm-scala-sdk[master]$ git status
On branch master
Your branch is ahead of 'origin/master' by 1 commit.
  (use "git push" to publish your local commits)
nothing to commit, working tree clean
```

Comment (kyuksel) [reply build.sbt:10]: No worries at all. I just thought perhaps we were bumping versions off repo for the time being or something.

File: src/main/scala/org/broadinstitute/dsp/HelmAlgebra.scala (hunk 4 above)
Comment (kyuksel) [reply src/main/scala/org/broadinstitute/dsp/HelmAlgebra.scala:13]: Yea, I thought about the same thing and decided to be explicit given the use case today. If that changes, we can modify the API.

**Pull request merged**